examples/scenarios/mesh/README.md

```diff
@@ -1,15 +0,0 @@
-# Mesh Example
```

Comment:
@mheap do you know why this is removed? (I just run `make speakeasy`)

Reply:
https://github.com/Kong/terraform-provider-konnect-beta/blob/main/Makefile#L6

Please remove this line and try again. This was committed by mistake when testing the resource rename script

https://github.com/Kong/terraform-provider-konnect-beta/commit/f415cc2a39987fc923921a752a171192b982dcb9

Reply:
I've done that but that did not help https://github.com/Kong/terraform-provider-konnect-beta/pull/7/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52L6

Reply:
Closing the loop. Fixed in https://github.com/Kong/terraform-provider-konnect-beta/pull/12